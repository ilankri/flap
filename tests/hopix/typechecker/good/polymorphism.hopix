fun id ['a] (x : 'a) : 'a = x

val y : int = id [int] (42)