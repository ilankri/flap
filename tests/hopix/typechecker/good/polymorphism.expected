id : 'a -> 'a
y : int
